Source/Core/ClippingPlane.js

```diff
@@ -0,0 +1,155 @@
+define([
+        './Cartesian3',
+        './Check',
+        './defined',
+        './defineProperties'
+    ], function(
+        Cartesian3,
+        Check,
+        defined,
+        defineProperties) {
+    'use strict';
+
+    /**
+     * A Plane in Hessian Normal form to be used with ClippingPlaneCollection.
```

Comment:
Use `{@link ClippingPlaneCollection}`.

Check out the [Documentation Guide](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/DocumentationGuide) if you need a refresher on best practices.